XorSwap/C++/xorswap.cpp

```diff
@@ -0,0 +1,17 @@
+#include <bits/stdc++.h>
```

Comment:
Please remove this header. This header is quite linux specifc and only support on the ```gcc```. Additional it increase the compile time.